spreadsheet/sharedstrings.go

```diff
@@ -57,6 +58,15 @@ func (s SharedStrings) GetString(id int) (string, error) {
 	si := s.x.Si[id]
 	if si.T != nil {
 		return *si.T, nil
+	} else if l := len(si.R); l > 0 {
+		if l == 1 {
+			return si.R[0].T, nil
+		}
+		var parts []string
```

Comment:
I would suggest making a slice with the length of len(si.R) right away and get rid of append, to avoid reallocations during the loop itself, to make this more efficient. Also this behavior can be achieved making the slice of 0-length but cap=len(si.R) and still using append. Both ways should behave equally